src/cloudai/schema/test_template/chakra_replay/slurm_command_gen_strategy.py

```diff
@@ -14,21 +14,12 @@
 
 from typing import Any, Dict, List
 
-from cloudai._core.system import System
 from cloudai.schema.system.slurm.strategy import SlurmCommandGenStrategy
 
 
 class ChakraReplaySlurmCommandGenStrategy(SlurmCommandGenStrategy):
     """Command generation strategy for ChakraReplay on Slurm systems."""
 
-    def __init__(
```

Comment:
Thanks for cleaning up. Just a quick question: Is it part of "Check that _core doesn't depend on other parts"? Did you add these changes to this commit for convenience?

Reply:
Removal of the constructors wasn't necessary, I was checking the dependencies to ensure my change is safe. At some point I realised, that these constructors could be removed and decided to include it into the same PR. If you think it will be better to split, I can do that too.

Reply:
Let's have it in the same PR. Thanks.